pages/ksphere/konvoy/latest/download/index.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: layout.pug
+navigationTitle: Download
+title: Download Konvoy
+menuWeight: 2
+excerpt: Learn how to download an instance of Konvoy 
```

Comment:
I suggest removing the words "an instance of"?  The download is for the Konvoy CLI that is used to install/deploy/manage Konvoy clusters.  Konvoy also refers to a distribution of kubernetes.